plugins/Speech/__init__.py

```diff
@@ -75,10 +75,25 @@
 )
 
 
-import win32com
-from time import strftime
-import threading
-import pythoncom
+import win32com # NOQA
+import threading # NOQA
+import pythoncom # NOQA
+from time import strftime # NOQA
+
+
+def _GetDescInvokeType(entry, invoke_type):
```

Comment:
Maybe we should put those function in a separate file, something like eg.patches or eg.compatibility, in the case there will be more of those bugs? (dde?)